v2/emailpassword/nextjs/next-steps.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: next-steps
-title: 6. Next steps
```

Comment:
why has this file changed? it's part of the pages dir.

Reply:
Fixed! Both pages and app dir were copying the same file. Using a different file for app dir docs now.